src/utils/timer.js

```diff
@@ -1,2 +1,2 @@
 // @flow
-export const sleep: Function = (ms: number) => new Promise((resolve) => setTimeout(resolve, ms))
+export const sleep: Function = (ms: number) => new Promise(resolve => setTimeout(resolve, ms))
```

Comment:
Prettier 2.0 forces brackets around function arguments. Could you please run `prettier:fix` and also make sure that your branch is upgraded?